src/Camera.jl

```diff
@@ -7,6 +7,7 @@ mutable struct Camera
     position::Vector2f
     startingCoordinates::Vector2f
     target
+    windowPos::Vector2
```

Comment:
Come up with a better name. This is the position of the window that contains this in the editor.